pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/SampleIdTestUtil.java

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.dsm.util;
+
+import org.broadinstitute.dsm.db.dao.ddp.tissue.TissueSMIDDao;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SampleIdTestUtil {
+
+    private final TissueSMIDDao smIdDao = new TissueSMIDDao();
+
+    private final Set<Integer> createdSampleIds = new HashSet<>();
+
+    public int createSampleForTissue(int tissueId, String user, String sampleType, String sampleName) {
+        int sampleId = smIdDao.createNewSMIDForTissue(tissueId, user, sampleType, sampleName);
+        createdSampleIds.add(sampleId);
+        return sampleId;
+    }
+
+    public void deleteCreatedSamples() {
```

Comment:
There's probably some generalized pattern lurking here--our test utils create rows and then have various methods to delete the things that were created, usually called within a teardown method.

Reply:
I can try to change this use the util I created